src/thriveopedia/fossilisation/FossilisedSpecies.cs

```diff
@@ -0,0 +1,243 @@
+    public const string SAVE_INFO_JSON = "info.json";
+
+    /// <summary>
+    ///   Name of this saved species on disk.
+    /// </summary>
+    public string Name { get; set; } = "invalid";
+
+    /// <summary>
+    ///   General information about this saved species.
+    /// </summary>
+    public FossilisedSpeciesInformation Info { get; set; } = new();
+
+    /// <summary>
+    ///   The species to be saved/loaded.
+    /// </summary>
+    public Species Species { get; set; } = null!;
+
+    /// <summary>
+    ///   Creates a list of existing fossilised species names to prevent unintended overwrites.
+    /// </summary>
+    /// <returns>A list of names of .thrivefossil files in the user's home directory</returns>
+    public static List<string> CreateListOfSaves()
+    {
+        var result = new List<string>();
+
+        using (var directory = new Directory())
+        {
+            if (!directory.DirExists(Constants.FOSSILISED_SPECIES_FOLDER))
+                return result;
+
+            directory.Open(Constants.FOSSILISED_SPECIES_FOLDER);
+            directory.ListDirBegin(true, true);
+
+            while (true)
+            {
+                var filename = directory.GetNext();
+
+                if (string.IsNullOrEmpty(filename))
+                    break;
+
+                if (!filename.EndsWith(Constants.FOSSIL_EXTENSION, StringComparison.Ordinal))
+                    continue;
+
+                // Skip folders
+                if (!directory.FileExists(filename))
+                    continue;
+
+                result.Add(filename);
+            }
+
+            directory.ListDirEnd();
+        }
+
+        using var file = new File();
+        result = result.OrderBy(item =>
+            file.GetModifiedTime(Path.Combine(Constants.FOSSILISED_SPECIES_FOLDER, item))).ToList();
+
+        return result;
+    }
+
+    /// <summary>
+    ///   Loads a fossilised species by its filename.
+    /// </summary>
+    /// <param name="saveName">The name of the .thrivefossil file (including extension)</param>
+    /// <returns>The species saved in the provided file</returns>
+    public static Species LoadSpeciesFromFile(string saveName)
+    {
+        var target = Path.Combine(Constants.FOSSILISED_SPECIES_FOLDER, saveName);
+        var (_, species) = LoadFromFile(target);
+
+        return species;
+    }
+
+    /// <summary>
+    ///   Saves this species to disk.
+    /// </summary>
+    public void SaveToFile()
+    {
+        if (Species is not MicrobeSpecies)
+        {
+            throw new NotImplementedException("Saving non-microbe species is not yet implemented");
+        }
+
+        // For now, save everything as a microbe to aid deserialization
+        var speciesInfo = new FossilisedSpeciesInformation
+        {
+            Type = FossilisedSpeciesInformation.SpeciesType.Microbe,
+        };
+
+        WriteRawSaveDataToFile(speciesInfo, Species.StringCode, Name + Constants.FOSSIL_EXTENSION_WITH_DOT);
```

Comment:
Is there a reason why the image is not saved in the fossil. I think it would make this much more helpful format if the screenshot of the thing would also be saved. That'd make the museum load much faster, right?

Reply:
The reason is I didn't know how to. Just tried it and it would take a significant refactor of the architecture, so I don't think it's worth it for now. There are other, easier ways to get better performance with the museum (like only loading when the Thriveopedia is opened, not when the museum page is).

Reply:
Doesn't the fossilization dialog show a preview image of the species? That can be grabbed quite easily I think. I'll give that a go with the other saving fixes I'll try to do today.

Reply:
I've now done this and about to commit. I had to extend the species preview system a bit to get it to save the Images when needed for writing into the fossilized files.